PLATFORMS.md

```diff
@@ -54,6 +54,7 @@ In this policy, the words "must" and "must not" specify absolute requirements th
 - x86_64/amd64/x64 for Windows 2022
 - armeabi-v7a, arm64-v8a, x86, x86_64 for Android
 - aarch64 for Apple iOS and tvOS (CMake `-DPLATFORM=OS64` and `TVOS`)
+- riscv32, riscv64 for Zephyr
```

Comment:
My suggestion then would be to add "Zephyr" to all HW platforms listed above and supported by Zephyr (possibly incl. "x64" as per separate comment).

Reply:
made change to platforms